warp/yul/utils.py

```diff
@@ -181,11 +182,25 @@ def check_installed_solc(source_version: float) -> str:
     return src_ver
 
 
-def cairoize_bytes(bs: bytes) -> tuple(list[int], int):
+def cairoize_bytes(bs: bytes) -> tuple(List[int], int):
     """Represent bytes as an array of 128-bit big-endian integers and
     return a number of unused bytes in the last array cell.
     """
     unused_bytes = -len(bs) % 16
     bs = bs.ljust(len(bs) + unused_bytes, b"\x00")
     arr = [int.from_bytes(bs[i : i + 16], "big") for i in range(0, len(bs), 16)]
     return (arr, unused_bytes)
+
+
+def make_abi_StarkNet_encodable(abi):
+    """
+    Because we need 31 byte addresses for starknet, we need to encode them as uint256,
+    but if we pass an abi with address types to solcx, it will try to validate the address by
+    checksum & making sure it's 20-bytes. So we just change the address abi types to uint256.
+    This is fine because our fork of solc treats address types as uint256 anyway.
+    We will replace this hackery with our own encoder in the future.
+    """
+    lol = f"{abi}"
```

Comment:
`lol` can be inlined. Also, I might be not fun, but I'd prefer to not have such variable names =)